docs/UserGuide.md

```diff
@@ -39,136 +39,281 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
-
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
-</div>
-
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
-
-
-### Adding a person: `add`
-
-Adds a person to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
-
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
-
-### Listing all persons : `list`
-
-Shows a list of all persons in the address book.
-
-Format: `list`
-
-### Editing a person : `edit`
-
-Edits an existing person in the address book.
+---
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+### Feature 2: Add New Customer
+
+**Purpose:**  
+This feature allows you to enter and save detailed records for new customers. Each customer's record includes their name, contact number, email, occupation, and income.
+
+**How to Use It:**  
+- **Command Format:** 
+```
+add --name <customer name> --contact <customer contact> --email <customer email> --job <job name> --income <customer income>
+ ```
+- **Example:** 
+```
+add --name TAN LESHEW --contact 66997788 --email mrtan@ntu.com --job doctor of computer science --income 1000000000
+```
+
+#### Parameters
+| Parameter | Expected Format                              | Explanation                                                |
+|-----------|----------------------------------------------|------------------------------------------------------------|
+| name      | Alphanumeric, capitalized                    | Names are in block letters for clarity and consistency.    |
+| contact   | 8-digit number, starts with 8 or 9           | Ensures the contact number is valid in Singapore.          |
+| email     | Must include "@" and domain, case insensitive| Verifies that the email address is in a standard format.   |
+| job       | Any text, case insensitive                   | Accepts all job titles without case sensitivity.           |
+| income    | Non-negative integers                        | Only positive numbers or zero are valid for income fields. |
+
+#### What to Expect
+- **If Successful:** You'll see a message: "New Contact added. Their ID is `<new customer ID>`."
```

Comment:
Since we are probably keeping their way of representing a contact and we are removing unique customer ids, this should be changed to `New person added: <NAME>; Contact: <CONTACT NUMBER>; Email: <EMAIL>; Job: <JOB>;  Income: <INCOME>; Tags: <TAGS>`

Reply:
Since Tag can be assigned only by the `tag` command, in add new customer case, should we just go by the Tag field N/A or just not show (cuz in use case the steps should be add new customer > the agent analyze > decide the tier)